networkx/algorithms/dag.py

```diff
@@ -1223,37 +1224,86 @@ def dag_to_branching(G):
 
 @not_implemented_for("undirected")
 @nx._dispatchable
-def compute_v_structures(G):
-    """Iterate through the graph to compute all v-structures.
+def compute_colliders(G):
+    """In a DAG, if you have three nodes A, B, and C, and there are edges from A to C
+    and from B to C, then C is a collider. This means that both A and B are "causing" C
+    in the graph, and conditioning on C would open up an association between A and B.
```

Comment:
The first line of a doc_string is used to describe the hyperlink in the docs for that function. So it should be a short (one-line) description followed by a blank line and then include this paragraph.  

Something like:
```suggestion
    """Yields 3-tuples of nodes that represent colliders in G
    
    In a DAG, if you have three nodes A, B, and C, and there are edges from A to C
    and from B to C, then C is a collider. This means that both A and B are "causing" C
    in the graph, and conditioning on C would open up an association between A and B.
```